docs/source/install/vagrant.rst

```diff
@@ -0,0 +1,121 @@
+.. |br| raw:: html
+
+   <br />
+
+
+Vagrant & Virtual Appliance
+============================
+Vagrant / OVA is a quick and easy way to try |st2| in a VM.
+
+It's already pre-installed, configured and tested virtual appliance image and so saves time avoiding
+time-consuming installation and configuration steps. Perfect for a quick platform overview,
+testing, demo or even using |st2| in isolated from the internet air-gapped systems.
+
+We highly recommend to use a Vagrant box to get familiar with the |st2| platform.
+
+.. contents:: Contents
+   :local:
+
+The source code is available as a GitHub repo:
+`StackStorm/packer-st2 <https://github.com/StackStorm/packer-st2>`_
+
+---------------------------
+
+Requirements
+------------
+`Vagrant <https://www.vagrantup.com/>`_ and `Virtualbox <https://www.virtualbox.org/>`_ are required.
+If you're not familiar with Vagrant, we recommend to take a look at `Introduction to Vagrant <https://www.vagrantup.com/intro/index.html>`_.
+
+Quick Start
+-----------
+Starting a |st2| Vagrant VM is easy:
+
+.. code-block:: bash
+
+  vagrant init stackstorm/st2
+  vagrant up
+  vagrant ssh
+
+
+Virtual Appliance / OVA
+-----------------------
+As alternative to Vagrant box is Virtual appliance which is available for download as ``.OVA``
+image from the `StackStorm/packer-st2 Github Releases <https://github.com/StackStorm/packer-st2/releases>`_
+page. It might be especially helpful for the isolated from the internet air-gapped environments.
+
+.. note::
+
+    *Linux login credentials:* |br|
+    Username: ``vagrant`` |br|
+    Password: ``vagrant`` |br|
+
+    *StackStorm login details:* |br|
+    Username: ``st2admin`` |br|
+    Password: ``Ch@ngeMe`` |br|
+
+.. warning::
+
+    If using OVA in production environment, don't forget to change the default credentials
+    and delete SSH authorized keys for ``vagrant`` linux user.
+
+
+At the moment only Virtualbox provider is supported. VMWare-compatible virtual appliance is
+available with `StackStorm Enterprise (EWC) <https://stackstorm.com/#product>`_ image.
+Ask `StackStorm Support <support@stackstorm.com>`_ for more info.
+
+
+Tips & Tricks
+-------------
+Updating the Vagrant box
+~~~~~~~~~~~~~~~~~~~~~~~~
+Once the newer box version is released, Vagrant will warn you about the available update.
+To update the box:
+
+.. code-block:: bash
+
+    vagrant box outdated
+    vagrant box remove stackstorm/st2
```

Comment:
Is there any particular reason you're not using `vagrant box update`?

Reply:
Good catch, let me try that.
It wasn't a thing before.

Reply:
And so the advantages with `vagrant box remove` Vagrant will ask you to destroy the already running boxes:
```
$ vagrant box remove centos/7
Removing box 'centos/7' (v1803.01) with provider 'virtualbox'...
Box 'centos/7' (v1802.01) with provider 'virtualbox' appears
to still be in use by at least one Vagrant environment. Removing
the box could corrupt the environment. We recommend destroying
these environments first:

default (ID: dddd6bdb40794e41bcc98071b15f8ec7)

Are you sure you want to remove this box? [y/N] y
```

where user has plenty of control.
And this way next `vagrant up` starts with the updated version, instead of running the older one.

Reply:
Okay, this makes sense. Thanks for the explanation.  👍